### PR TITLE
WIP: Feature/recharts

### PR DESCRIPTION
**Pull request by mluena**
@mbarrenechea 
  
This is the config file I would like you to look at:
app/javascript/components/widget/widget-templates/number-of-services/config.js

### CODE REVIEW

File: app/javascript/components/widget/widget-templates/points-percentage/component.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import Chart from 'components/recharts';
+
+import config from './config';
+
+const AccessPointsPercentage = ({ widgetData }) => {
+
+
+  if (!widgetData) return null;
+
+  const data = config.parse(widgetData);
```
Comment (mbarrenechea) [inline app/javascript/components/widget/widget-templates/points-percentage/component.js]: there is one thing I don't like about this approach. You are parsing the data on every render... It's up to you to change it

**Comment by mbarrenechea** [comment]
BTW, Nice work

File: app/javascript/components/sidebar/sidebar-analysis/component.js [hunk 2]
```diff
@@ -8,7 +8,8 @@ import MenuItem from 'components/sidebar/menu-item';
 import Nearby from 'components/sidebar/nearby';
 import AreaOfInterest from 'components/sidebar/area-of-interest';
 import Jurisdiction from 'components/sidebar/jurisdiction';
-import AnalysisResult from 'components/sidebar/analysis-result';
+import AnalysisResult from 'components/widget/analysis-result';
+import WidgetWrapperComponent from 'components/widget';
```
[the remark this answers is not in the document]
Comment (mluena) [reply app/javascript/components/sidebar/sidebar-analysis/component.js]: I haven't clean anything yet

File: app/javascript/components/widget/widget-templates/points-percentage/component.js (hunk 1 above)
Comment (mluena) [reply app/javascript/components/widget/widget-templates/points-percentage/component.js]: siiiiigh

**Pull request merged**